### ISSUE TITLE
Long video loading in playlist

### ISSUE THREAD

**Issue by Albatrosicks** [silverwind/droppy#286]
When you're switching a lot of videos, you can get a video which will load extremely long time.
So, I guess it's happening because previous videos weren't completely loaded.
https://puu.sh/tnozx/e1b7e0cd3b.png

**Comment by silverwind** [comment]
Do you perhaps see any errors when it happens and you open the browser console?

**Comment by Albatrosicks** [comment]
No. It's just still loading and I can't load it in new tab until connection(tab) will be closed.

**Comment by silverwind** [comment]
v6.2.0 uses a more new video player. Maybe you can give it a try.

**Comment by Albatrosicks** [comment]
yay, now i got an error
```
2017-03-30 01:39:23 [ERROR] "start" option must be <= "end" option
    at new ReadStream (fs.js:1853:13)
    at ReadStream (...\AppData\Roaming\npm\node_modules\droppy\node_modules\graceful-fs\graceful-fs.js:178:28)
    at Object.createReadStream (...\AppData\Roaming\npm\node_modules\droppy\node_modules\graceful-fs\graceful-fs.js:220:12)
    at streamFile (...\AppData\Roaming\npm\node_modules\droppy\server\server.js:1278:10)
    at ...\AppData\Roaming\npm\node_modules\droppy\server\server.js:928:9
    at ...\AppData\Roaming\npm\node_modules\droppy\node_modules\graceful-fs\polyfills.js:287:18
    at FSReqWrap.oncomplete (fs.js:114:15)
```

**Comment by silverwind** [comment]
That error should hopefully be gone as of https://github.com/silverwind/droppy/commit/5b9f39f9a7449155bfbf3312f416f04e6f587959, and if not, more details should be logged. I'll release that after a few more tests.

**Comment by silverwind** [comment]
Video issues should be fixed as of 6.3.7. Can you retest?

**Comment by Albatrosicks** [comment]
It seems fixed. I tried to play 0.5TB playlist and got a few lags which were a bit inconsequential
